ecommerce_integrations/unicommerce/order.py

```diff
@@ -48,25 +48,34 @@ def sync_new_orders(client: UnicommerceAPIClient = None, force=False):
 
 	status = "COMPLETE" if settings.only_sync_completed_orders else None
 
-	new_orders = _get_new_orders(client, status=status)
-
+	new_orders = _get_new_orders(client, status=status, check=True)
 	if new_orders is None:
 		return
-
 	for order in new_orders:
 		sales_order = create_order(order, client=client)
+		# if settings.only_sync_completed_orders:
+		#     _create_sales_invoices(order, sales_order, client)
```

Comment:
Why is this just removed? Again, don't just break existing things because multiple people use it.

Reply:
Reverted

Reply:
I've added test cases, it would be great if you could go over them.